### ISSUE TITLE
FlyingKinect world wrong definition

### ISSUE THREAD

**Issue by fqez** [RoboticsURJC/JdeRobot#245]
Hi, 

the flyingKinect.world is wrong defined as it includes models outside the world tag, so instead this:

```
<?xml version="1.0" ?>
<sdf version="1.4">
    <include>
      <uri>model://ground_plane</uri>
    </include>
    <include>
      <uri>model://flyingKinect</uri>
    </include>
  <world name="default">
    <include>
      <uri>model://sun</uri>
    </include>
  </world>
</sdf>
```

we need this:

```
<?xml version="1.0" ?>
<sdf version="1.4">
  <world name="default">
    <include>
      <uri>model://ground_plane</uri>
    </include>
    <include>
      <uri>model://flyingKinect</uri>
    </include>
    <include>
      <uri>model://sun</uri>
    </include>
  </world>
</sdf>
```

So if we launch the first one, it does not load the flyingKinect model, only an empty world.

**Comment by varhub** [comment]
It raises a warning like "include tag was not expected, skipping it", or similar?

**Comment by fqez** [comment]
No, it does not say anythig. It's also true that I didn't launch the world with the 'verbose' flag of Gazebo. I will tell you if something like that appear in the log files soon.

**Comment by fqez** [comment]
Fix included in #270
